mule-artifact-it/mule-deployer-it/src/test/java/integrationTests/mojo/CloudHubDeploymentTest.java

```diff
@@ -56,13 +55,14 @@ public void before() throws VerificationException, InterruptedException, IOExcep
     verifier.setEnvironmentVariable("username", System.getProperty("username"));
     verifier.setEnvironmentVariable("password", System.getProperty("password"));
     verifier.setEnvironmentVariable("environment", "Production");
-    verifier.setEnvironmentVariable("mule.version", System.getProperty("mule.version"));
+    verifier.setEnvironmentVariable("mule.version", "4.0.0-FD");
```

Comment:
please create a jira so we don't forget to change this when CH is rename the version and add a comment with the jira number